content/deploy-troubleshooting.md

```diff
@@ -24,9 +24,9 @@ Note that code-level review lies outside the scope of Galaxy's support. If this
 
 <h2 id="502-errors">502 errors</h2>
 
-You may see a 502 error with the message `Registered endpoints failed to handle the request` when you try to visit your URL. This means that the request failed, despite the fact that our system thought there was a healthy container at the beginning of the request.
+You may see a 502 error with the message `Registered endpoints failed to handle the request` when you try to visit your URL. This means that the request failed, despite the fact that your system thought there was a healthy container at the beginning of the request.
```

Comment:
No, the system that is checking for container health and the system that is building containers is "our system".  But the container that is failing to respond is their backend (usually one word too).

Also I don't think 502s are usually from deployment failure.  503 can be from deployment failure.  502 means that the proxy knew about a running healthy container, tried to talk to it, and then something failed.